### ISSUE TITLE
Add support for math operations

### ISSUE THREAD

**Issue by nguyenquyhy** [nguyenquyhy/Flight-Tracker-StreamDeck#224]
- [ ] Feedback value (e.g. FUEL_TANK_LEFT_MAIN_QUANTITY*6.7) #199
- [ ] Toggle parameter (e.g. PLANE HEADING DEGREES MAGNETIC - 4294967295 / 8) #223

**Comment by RonPi** [comment]
I created an Angle of Attack button. Right now I have the limits at 0 on the right and 10 degrees* on the left. Alas, the units returned are in radians, and I think in degrees. Also, counterintuitively, an increased angle of attack yields smaller numbers. EG: level flight yields 3.14......... One degree would be 3.106......... Zero would be you are falling straight down. I would like to do as stated above add math to the end of "ANGLE OF ATTACK INDICATOR" to convert it to degrees.

This could be done as shown, or have an additional entry for "Math applied to value" or some such.

* I am going to add a 2nd one with limits of Pi (0 degrees) and 3.072 (2 degrees). I find flying close to zero degrees yields the best range. I expect the exact speed and AoA will probably vary by plane. If you go faster, skin friction will rob you, and the torque to overcome it will cost you range. Perhaps just arriving at ZERO degrees and going no faster would do the job. You can't just pick a speed, as it also depends on the load you are carrying. Faster for a heavier load. Too great an angle of attack and you are flying uphill. I have done charts on this, but on the LearJet 45 I found that quite tedious when I noted their angle of attack gauge always read  0.1 degrees when I found the most economical speed, so I quit making the charts and just followed the gauge... for ascent, cruise and descent.

**Comment by nguyenquyhy** [comment]
Have you tried to put Unit as Degree, and put Minimum value > Maximum value in the gauge?

**Comment by RonPi** [comment]
I did the min and max part. I just quit flying because my butt got sore, but if you think saying DEGREES, DEGREE OR DEG after the ANLGE OF ATTACK will work, I guess I could give it a try.

**Comment by nguyenquyhy** [comment]
No, there is an input field named "Unit" in the plugin. You can fill in "Degree" in there.

**Comment by RonPi** [comment]
Thanks, I will check that out.
